### PR TITLE
2257 suburban junction infographics

### PR DESCRIPTION
**Pull request by ziv17**
This PR shows the 3 all-location widgets for SuburbanJunction locations. For example, the following URL works:
1. `api/infographics-data?news_flash_id=18264&years_ago=3&lang=he`
2. `api/infographics-data?road1=5&road2=40&years_ago=8&lang=he`
3. `api/infographics-data?road1=40&road2=5&years_ago=8&lang=he`
4. `api/infographics-data?non_urban_intersection=4021&years_ago=8`
5. `api/infographics-data?non_urban_intersection_hebrew=מחלף ירקון&years_ago=3`

NOTE:
Currently URLs 3 and 4 shows different accidents because `road1` is compared to `road1` in the accident (same for `road2`). 
In the next PR I will change the filter to take all accidents that their roads appear in the junction roads (There are junctions with more than two roads).

### CODE REVIEW

File: anyway/config.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 SQLALCHEMY_TRACK_MODIFICATIONS = True
 ENTRIES_PER_PAGE = os.environ.get("ENTRIES_PER_PAGE", 1000)
 SQLALCHEMY_ENGINE_OPTIONS = {}
-# SQLALCHEMY_ECHO = True
+SQLALCHEMY_ECHO = True
```
Comment (BarVolunteering) [inline anyway/config.py:15]: Needed?

File: translations/he/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -339,141 +347,147 @@ msgstr ""
...
-#: anyway/widgets/suburban_widgets/street_view_widget.py:25
+#: anyway/widgets/road_segment_widgets/street_view_widget.py:25
 msgid "Street view widget"
 msgstr "Street view widget"
 
-#: anyway/widgets/suburban_widgets/suburban_crosswalk_widget.py:72
+#: anyway/widgets/road_segment_widgets/suburban_crosswalk_widget.py:72
 #: anyway/widgets/urban_widgets/urban_crosswalk_widget.py:75
-msgid "Pedestrian injury comparison on "
+#, fuzzy
```
Comment (BarVolunteering) [inline translations/he/LC_MESSAGES/messages.po:473]: Remove fuzzy?

File: translations/he/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -339,141 +347,147 @@ msgstr ""
...
 msgstr "פגיעות בהולכי רגל על"
 
-#: anyway/widgets/suburban_widgets/top_road_segments_accidents_per_km_widget.py:88
+#: anyway/widgets/road_segment_widgets/top_road_segments_accidents_per_km_widget.py:88
 msgid "Segments with most accidents per Km"
 msgstr "מקטעים עם הכי הרבה תאונות לק\"מ "
 
-#: anyway/widgets/suburban_widgets/top_road_segments_accidents_per_km_widget.py:94
+#: anyway/widgets/road_segment_widgets/top_road_segments_accidents_per_km_widget.py:89
+#, fuzzy
```
Comment (BarVolunteering) [inline translations/he/LC_MESSAGES/messages.po:482]: Remove fuzzy?

File: translations/he/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -483,9 +497,20 @@ msgstr "תאונות הולכי רגל לפי חומרה ולפי שנה"
 
 #: anyway/widgets/urban_widgets/severe_fatal_count_by_vehicle_by_year_widget.py:98
 #: anyway/widgets/urban_widgets/small_motor_severe_fatal_count_by_year_widget.py:60
-msgid "Severe or fatal accidents on bikes, e-bikes, or scooters in "
+#, fuzzy
```
Comment (BarVolunteering) [inline translations/he/LC_MESSAGES/messages.po:500]: Remove fuzzy?

File: translations/he/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -483,9 +497,20 @@ msgstr "תאונות הולכי רגל לפי חומרה ולפי שנה"
...
+msgid "Severe or fatal accidents on bikes, e-bikes, or scooters"
 msgstr "רוכבי אופנים, אופנים חשמליים, וקורקינט חשמלי שנפצעו קשה או נהרגו ב"
 
+#: anyway/widgets/urban_widgets/severe_fatal_count_by_vehicle_by_year_widget.py:99
+#: anyway/widgets/urban_widgets/small_motor_severe_fatal_count_by_year_widget.py:61
+msgid "yishuv_name"
+msgstr ""
+
+#: anyway/widgets/urban_widgets/urban_crosswalk_widget.py:76
+#, fuzzy
```
Comment (BarVolunteering) [inline translations/he/LC_MESSAGES/messages.po:510]: Remove fuzzy?

File: anyway/config.py (hunk 1 above)
Comment (ziv17) [reply anyway/config.py:15]: No... I use it for debugging and forgot to remove.

File: anyway/parsers/__init__.py [hunk 6]
```diff
@@ -5,7 +5,7 @@
     "רחוב": ["yishuv_name", "street1_hebrew"],
     "צומת עירוני": ["yishuv_name", "street1_hebrew", "street2_hebrew"],
     "כביש בינעירוני": ["road1", "road_segment_name"],
-    "צומת בינעירוני": ["road1", "road_segment_name", "road2", "non_urban_intersection_hebrew"],
+    "צומת בינעירוני": ["non_urban_intersection", "non_urban_intersection_hebrew", "road1", "road2"],
```
[the remark this answers is not in the document]
Comment (ziv17) [reply anyway/parsers/__init__.py:8]: One, we currently do not support suburban junction. And beyond that, I do not think this changes the output (i.e. the widget items)

File: translations/he/LC_MESSAGES/messages.po (hunk 3 above)
Comment (ziv17) [reply translations/he/LC_MESSAGES/messages.po]: Yes!

File: translations/he/LC_MESSAGES/messages.po (hunk 4 above)
Comment (ziv17) [reply translations/he/LC_MESSAGES/messages.po]: Yes!

File: translations/he/LC_MESSAGES/messages.po (hunk 5 above)
Comment (ziv17) [reply translations/he/LC_MESSAGES/messages.po]: Yes!

**Review by BarVolunteering — APPROVED** [review]
Looks good!

**Pull request merged**